requirements.in

```diff
@@ -46,4 +46,6 @@ opentelemetry-instrumentation-celery>=0.45b0
 opentelemetry-sdk>=1.24.0
 django-admin
 google-cloud-pubsub
-openai
\ No newline at end of file
+openai
+zipp>=3.19.1
+certifi>=2024.07.04
```

Comment:
I think if you merge https://github.com/codecov/shared/pull/286 and update shared then certify will be fixed here too